### PR TITLE
YugaV51 || Changes for text highlighting bugs

### PR DESCRIPTION
**Pull request by sonali1404**

### CODE REVIEW

File: src/main/java/com/twelfthmile/yuga/Yuga.java [hunk 1]
```diff
@@ -30,7 +30,7 @@ public static void init() {
         RootTrie root = getRoot();
     }
 
-    private static RootTrie getRoot() {
```
Comment (vizsatiz) [inline src/main/java/com/twelfthmile/yuga/Yuga.java:33]: Why do you make this public ?

File: src/main/java/com/twelfthmile/yuga/Yuga.java [hunk 2]
```diff
@@ -1295,6 +1297,25 @@ private static void setIfNumRange(String str, int i, FsaContextMap map) {
         }
     }
 
+    private static String checkIfData(String str, int j, FsaContextMap map) {
+        String sData = "";
+        switch (str.charAt(j)){
+            case 'k':
+                map.setVal("data_type","KB");
+                sData = " KB";
```
Comment (vizsatiz) [inline src/main/java/com/twelfthmile/yuga/Yuga.java:1300]: You can return inside the `case` block and avoid `break` and `sData`

File: src/main/java/com/twelfthmile/yuga/Yuga.java (hunk 1 above)
Comment (sonali1404) [reply src/main/java/com/twelfthmile/yuga/Yuga.java]: Made it private.

**Review by saxena-saumittra — APPROVED** [review]

File: src/main/java/com/twelfthmile/yuga/Yuga.java (hunk 2 above)
Comment (sonali1404) [reply src/main/java/com/twelfthmile/yuga/Yuga.java:1298]: This method's return type was string earlier but now its void and I am not returning anything.

**Pull request merged**